packages/api/src/store/db.ts

```diff
@@ -103,6 +106,13 @@ export class DB {
       console.log("no replica url found, not using read replica");
     }
 
+    this.metricHistogram = new Histogram({
+      name: "pgsql_query_duration_seconds",
```

Comment:
Can we namespace the metric with `livepeer_api`? Following the pattern from the `livepeer-data` metrics (not merged yet tho, still on review), we could set up a `namespace: "livepeer"` and a `subsystem: "api"`, WDYT? 

If the `prom-client` node lib does not have the concepts of namespace and subsystem, you can just prefix the metric name with `livepeer_api_` instead, so the fully qualified name is 
```suggestion
      name: "livepeer_api_pgsql_query_duration_sec",
```
(also abbreviated `seconds`, just a nit feel free to keep it full)

Reply:
It doesn't have a concept for namespace or prefixes, and when initialising a metric type; you specify the full metric `name` value (as done here).

I'll update the name prefix to reflect the service name (same would be applicable for http response duration metric I guess).

For the suffix `sec`, [prometheus recommends][1] having full, pluralised unit in the metric name. If we do have a consensus for naming, it'd be quicker to follow; or we can start writing/discussing one for later usage.

  [1]: https://prometheus.io/docs/practices/naming/